src/realm/table.hpp

```diff
@@ -2014,6 +2103,25 @@ inline void Table::set_ndx_in_parent(size_t ndx_in_parent) noexcept
     }
 }
 
+template <class T>
+void Table::set_list(size_t c, size_t r, const std::vector<T>& list)
```

Comment:
Will this update the accessor? Consider a binding:

    RealmList<int> iList = obj.getIntList();
    // iList is {1, 4}
    iList.set([{2, 5, 7});
    iList.get(2); // 7 or fail?